addons/mail/static/src/discuss/call/common/call_participant_card.scss

```diff
@@ -33,6 +33,23 @@
 
 .o-discuss-CallParticipantCard-avatar:not(.o-minimized) {
     background-color: var(--o-discuss-CallParticipantCard-avatarBgColor, #{$o-gray-700});
+    &.o-is-remote-video {
```

Comment:
Nitpicking: I would use `camelCase` for part name and modifier name.
Reasoning: this is more cohesive in the name of part name and modifier name, similarly to the BEM with each part. And 2nd argument is more that `o-some-thing` is used in many places in odoo and the risk for collision is quite high...